package.json

```
@@ -13,10 +13,15 @@
     "url": "git+ssh://git@github.com/elo7/doc-amd.git"
   },
   "scripts": {
-    "lint": "eslint doc.js test/*",
-    "pretest": "npm run lint && npm run build",
-    "test": "bash test.sh",
-    "build": "uglifyjs doc.js -o doc.min.js --compress drop_console=true --mangle --screw-ie8"
+    "build": "uglifyjs doc.js -o doc.min.js --compress drop_console=true --mangle --screw-ie8",
+    "server": "http-server -p 8888",
+    "start:dev": "run-p -l start watch",
+    "start": "npm isntall && run-s -l build server",
```

Não seria melhor o `npm ci`?
```suggestion
    "start": "npm ci && run-s -l build server",
```

---

:hammer_and_pick: